client/src/Authorization/Login/Login.js

```diff
@@ -0,0 +1,113 @@
+import React, { Component } from "react";
+import { Redirect } from "react-router-dom";
+import { authService } from "../_auth-services/authentication";
+import "./Login.css";
+import bookImage from "../../pictures/books.png";
+
+class Login extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
```

Comment:
As far as I understand we will use the user entity (Dto class) in the whole application. Shouldn't we have a user entity that can be retrieved from local storage? 

Like this one:
user = { 
                 name: username,
                 token: response.token
       }

Reply:
First this object has to be created and added to localStorage. It is added in user login. If the user logs out, this object is removed.

Reply:
Not sure if I understand. Nevermind!